src/redux/checkout/tests/checkout.actions.test.js

```diff
@@ -241,7 +257,356 @@ describe('get prices from NovaPoshta API', () => {
+  });
+
+  it('value by key "payload[0].CITY_ID" of returned object should be equal to "10748"', () => {
+    expect(payload[0].CITY_ID).toEqual('10748');
+  });
+
+  it('value by key "payload[0].CITY_UA" of returned object should to be "Дніпро"', () => {
+    expect(payload[0].CITY_UA).toEqual('Дніпро');
+  });
+});
+
+describe('get cities from UkrPoshta API', () => {
+  let type;
+  const payload = { id: 584 };
+
+  beforeEach(() => {
+    type = GET_UKRPOST_CITIES;
+  });
+
+  it('should return object', () => {
+    expect(typeof getUkrPostDistricts()).toStrictEqual('object');
+  });
+
+  it('should return object with two key/value "type: GET_UKRPOST_CITIES"', () => {
+    expect(getUkrPostCities(payload)).toStrictEqual({
+      type: GET_UKRPOST_CITIES,
+      payload
+    });
+  });
+
+  it('should to be truthy', () => {
+    expect(getUkrPostCities(payload)).toBeTruthy();
+  });
+
+  it('value by key "type" of returned object should to be equal to "GET_UKRPOST_CITIES"', () => {
+    expect(getUkrPostCities(payload).type).toEqual(type);
+  });
+});
+
+// post offices from UkrPoshta action tests
+describe('set post offices from UkrPoshta action test', () => {
+  let type;
+  let setUPPostOfficesAction;
+
+  const payload = [
+    {
+      CITY_ID: '10748',
+      CITY_UA: 'Дніпро',
+      CITY_UA_TYPE: 'м.',
+      POSTTERMINAL: '1',
+      POSTOFFICE_ID: '12489',
+      POSTOFFICE_UA: '49000 Дніпро',
+      POSTCODE: '49000',
+      ISAUTOMATED: '1',
+      PHONE: '056-744-81-90, 050-160-97-27',
+      STREET_ID_VPZ: '448135',
+      STREET_UA_VPZ: 'просп. Дмитра Яворницького, 62'
+    },
+    {
+      CITY_ID: '10748',
+      CITY_UA: 'Дніпро',
+      CITY_UA_TYPE: 'м.',
+      POSTTERMINAL: '1',
+      POSTOFFICE_ID: '19192',
+      POSTOFFICE_UA: '49001 Дніпро',
+      POSTCODE: '49001',
+      ISAUTOMATED: '1',
+      PHONE: null,
+      STREET_ID_VPZ: '64624',
+      STREET_UA_VPZ: 'вул. Михайла Грушевського, 10'
+    }
+  ];
+
+  beforeEach(() => {
+    type = SET_UKRPOST_POSTOFFICES;
+    setUPPostOfficesAction = setUkrPostPostOffices(payload);
+  });
+
+  it('should return object', () => {
+    expect(typeof setUPPostOfficesAction).toStrictEqual('object');
+  });
+
+  it('should be truthy', () => {
+    expect(setUPPostOfficesAction).toBeTruthy();
+  });
+
+  it('value by key "type" of returned object should be equal to "SET_UKRPOST_POSTOFFICES"', () => {
+    expect(setUPPostOfficesAction.type).toEqual(type);
+  });
+
+  it('value by key "payload[0].CITY_ID" of returned object should be equal to "10748"', () => {
+    expect(payload[0].CITY_ID).toEqual('10748');
+  });
+
+  it('value by key "payload[0].CITY_UA" of returned object should be equal to "Дніпро"', () => {
+    expect(payload[0].CITY_UA).toEqual('Дніпро');
+  });
+
+  it('value by key "payload[0].POSTOFFICE_ID" of returned object should be equal to "12489"', () => {
+    expect(payload[0].POSTOFFICE_ID).toEqual('12489');
```

Comment:
also this toEqual params should be consts, not strings

Reply:
Makes no sense if we import the same data into two different variables and then compare them to be equal.
As an option, we can remove tests that check payload.